### PR TITLE
Changed text for #526: @expand-text is for AVT's also

### PR DESCRIPTION
**Pull request by eriksiegel**
Changed the wording for the expand-text attribute for #526.

This does not (yet) include mentioning our new p:inline-expand-text attribute which is another issue.

### CODE REVIEW

File: xproc/src/main/xml/specification.xml [hunk 1]
```diff
@@ -2238,23 +2238,22 @@ replaced.</para>
...
-<tag class="attribute">p:expand-text</tag>.</para>
+        <para>The <code>expand-text</code> attribute controls whether or not descendant text nodes
+            <emphasis>and</emphasis> attribute nodes within <tag>p:inline</tag> elements are designated as value
+          templates. On elements in the XProc namespace, the attribute is named <tag class="attribute"
+          >expand-text</tag>. On elements that are not in the XProc namespace, the attribute is named <tag
+            class="attribute">p:expand-text</tag>.</para>
 
 <para>If the <code>expand-text</code> attribute appears on more than one
-element among the ancestors of a text node, only the value on the nearest
+element among the ancestors of the <tag>p:inline</tag>, only the value on the nearest
```
[the remark this answers is not in the document]
Comment (eriksiegel) [reply xproc/src/main/xml/specification.xml]: You mean our special p:inline-expand-text attributes?

[the remark this answers is not in the document]
Comment (eriksiegel) [reply xproc/src/main/xml/specification.xml]: Maybe we have to combine this issue with #508 (the p:inline-expand-text attribute) in one go...

[the remark this answers is not in the document]
Comment (gimsieke) [reply xproc/src/main/xml/specification.xml]: Yes, probably. 
And we should reconsider paragraph after `<!-- feature="p-inline-no-nesting" -->`. (I think Norm removed it in his local repo.) It says that `p:expand-text` within inlined content is passed as an ordinary attribute and will not be removed. But as it stands now, there is no mention of `p:expand-text` being removed from inlined content.

[the remark this answers is not in the document]
Comment (eriksiegel) [reply xproc/src/main/xml/specification.xml]: All right. I'll close this pull request for now and will try to do a more thorough rewrite of this prose in the spec.

**Pull request closed without merging**